File: 02_Lenguajes-de-programación/02_Java/02_operadores.java
/* Operadores */
package Paquete;


public class Main {

    public static void main(String[] args) {

        // 1. Operadores Aritméticos

        System.out.println(1 + 3); // Suma
        System.out.println(4 - 2); // Resta
        System.out.println(10 * 3); // Multiplicacion
        System.out.println(30 / 3); // Division
        System.out.println(30 % 3); // Residuo o modulo

        // 2. Operadores Comparativos

        System.out.println(4 == 4); // Igual que
        System.out.println(4 != 4); // Diferente
        System.out.println(4 > 4); // Mayor
        System.out.println(4 < 4); // Menor
        System.out.println(4 >= 4); // Mayor igual que
        System.out.println(4 <= 4); // Menor igual que

        // 3. Operadores Lógicos

        // - Operador AND (&&)
        System.out.println(true && true); // Verdadero
        System.out.println(true && false); // Falso
        System.out.println(false && true); // Falso
        System.out.println(false && false); // Falso

        // - Operador OR (||)
        System.out.println(true || true); // Verdadero
        System.out.println(true || false); // Verdadero
        System.out.println(false || true); // Verdadero
        System.out.println(false || false); // Falso

        // - Operador NOT (!)
        System.out.println(!true); // Falso
        System.out.println(!false); // Verdadero
    }

}

File: 02_Lenguajes-de-programación/09_Rust/02_operadores.rs
fn main() {
    let numero_1 = 10;
    let numero_2 = 3;

    // 1. Operadores Aritméticos
    let suma = numero_1 + numero_2;
    let resta = numero_1 - numero_2;
    let multiplicacion = numero_1 * numero_2;
    let division = numero_1 / numero_2;
    let residuo = numero_1 % numero_2;

    // 2. Operadores Comparativos
    let mayor_que = numero_1 < numero_2;
    let menor_que = numero_1 > numero_2;

    // 3. Operadores Lógicos
    let and = true && false; // Operador AND
    let or = true || false; // Operador OR
    let not = !true; // Operador NOT

    // Impresión de datos:
    println!("Suma: {}", suma);
    println!("Resta: {}", resta);
    println!("Multiplicación: {}", multiplicacion);
    println!("Division: {}", division);
    println!("Residuo: {}", residuo);

    println!("10 es MAYOR que 3: {}", mayor_que);
    println!("10 es MENOR que 3: {}", menor_que);

    println!("AND: {}", and);
    println!("OR: {}", or);
    println!("NOT: {}", not);
}
